QueueITLib/QueueITEngine.m

```diff
@@ -176,9 +180,16 @@ -(void)showQueue:(NSString*)queueUrl targetUrl:(NSString*)targetUrl
 }
 
 -(void)tryEnqueue
+{
+    [IOSUtils getUserAgent:^(NSString * userAgent) {
+        [self tryEnqueueWithUserAgent:userAgent];
+    }];
+}
+
+-(void)tryEnqueueWithUserAgent:(NSString*)secretAgent
```

Comment:
On pourrait renommer ça en `userAgent` non ?

Reply:
Je préfère pas, afin d'éviter des problèmes de merges sur les futurs version Queue-IT.
Si jamais il y a une modification sur cette méthode de la part de Queue-It, j'ai bien peur que Git ait du mal à merge ou ne fasse pas le diff.
Un peu comme ce qui est arrivé à la semaine anniversaire avec les `hosts`